build.sbt

```diff
@@ -34,6 +34,7 @@ lazy val microservice = Project(appName, file("."))
     // Use the silencer plugin to suppress warnings
     // You may turn it on for `views` too to suppress warnings from unused imports in compiled twirl templates, but this will hide other warnings.
     scalacOptions += "-P:silencer:pathFilters=routes",
+    scalacOptions += "-P:silencer:globalFilters=Unused import",
```

Comment:
As discussed we prefer suppressing the warnings in views rather that suppressing Unused imports everywhere in the code, so could you update this, add views in line 36:
scalacOptions += "-P:silencer:pathFilters=views;routes"
and remove line 37 please

Reply:
done thanks